src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class ColorSupplier {
+    public static String getRandomColor() {
+        int randomIndex = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[randomIndex];
+        return color.toString();
```

Comment:
Use the name() method instead of toString() for getting the String representation of enum constants to avoid issues if toString() is overridden.